app/components/verkiezingen/kieslijst-splitter.hbs

```diff
@@ -158,16 +158,23 @@
       <div class="au-u-bold au-u-flex--inline">
         {{this.selectedKieslijst.lijstnaam}}
       </div>
-      in fracties
+      in
+      {{this.fractieText "de"}}
```

Comment:
didn't know you could already do this, feels more like something from polaris release. nice
but it means you need the action decorator right?

Reply:
I think you need the action decorator indeed, not sure though, was more of an assumption.

Reply:
yes you need it